login.php

```diff
@@ -16,17 +16,33 @@
     $accountID = mysqli_real_escape_string($db,$_POST['accountID']);
     $password = mysqli_real_escape_string($db,$_POST['password']);
 
-    $sql = "SELECT accountID FROM 1Account WHERE accountID = '$accountID' and password = '$password'";
+    $sql = "SELECT accountID,profileName FROM 1Account WHERE accountID = '$accountID' and password = '$password'";
     $result = mysqli_query($db,$sql);
     $row = mysqli_fetch_array($result,MYSQLI_ASSOC);
     $count = mysqli_num_rows($result);
+    $_SESSION['profileName']    = $row['profileName'];
```

Comment:
Should be setted with accountID on row 31, its not good to set Session variables if the login might not be valid